### PR TITLE
Docs/add readme

### PR DESCRIPTION
**Pull request by DiplomaZ**
Populated README with working-description and formatting. Integrated links to backend URL, including swagger. FE dummy links will be replaced after FE is deployed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,72 +2,58 @@
 
 For steps on how to work with this repository [please see here](https://docs.labs.lambdaschool.com/labs-spa-starter/)
 
-🚫 Note: All lines that start with 🚫 are instructions and should be deleted before this is posted to your first Pull-Request is merged. This is intended to be a guideline. Feel free to add your own flare to it.
-
 # Project
 
-You can find the deployed project at [🚫URL NAME GOES HERE](🚫copy and paste URL here).
```
Comment (lgv-0) [inline README.md:9]: Looks like we are live at https://labspt13-resfeber-b-be.herokuapp.com/

File: README.md [hunk 2]
```diff
@@ -2,72 +2,58 @@
...
-- feature three
-- feature four
-- feature five
+- Authenticate users via Okta token
+- Enable client to create, read, update, and delete profiles
+- Parse incoming form data through data-science API and return prediction-optimized travel itinerary
+- Enable client to save travel itineraries for future review (DS data will be retrieved accordingly)
 
-#### Back end deployed to `🚫insert service here`
+#### [Link to front end](#)
```
Comment (lgv-0) [inline README.md]: Forgot to link the FE-Repo here

**Review by lgv-0 — CHANGES_REQUESTED** [review]
Forgot a few links for deployment and FE Repo

File: README.md (hunk 2 above)
Comment (DiplomaZ) [reply README.md]: Good catch, thank you!

File: README.md (hunk 1 above)
Comment (DiplomaZ) [reply README.md:9]: I had a little confusion about whether or not we were linking the deployed full-product or just the BE, so thank you for clarifying that.

**Review by lgv-0 — APPROVED** [review]
LGTM

**Pull request merged**